source/_integrations/nut.markdown

```diff
@@ -22,6 +22,9 @@ The Network UPS Tools (NUT) integration allows you to monitor and manage a UPS (
 
 {% include integrations/config_flow.md %}
 
+## Connecting to Add-on instance
+If you are using [NUT add-on](https://github.com/hassio-addons/addon-nut) to run the server, you have to use the add-on specific hostname for the connection. It can be found on the add-on status page and looks like this: `a0d7b954-nut`.
```

Comment:
These are instructions for a third-party add-on, that is not supported by the Home Assistant project.

You can run this in many different ways, and how to run it, is not in scope of our documentation. The upstream documentation might be a more fitting place in that case.

../Frenck

Reply:
True, I didn't realize that.